### DISCUSSION TITLE
有支持 Taro3 Vue3 框架的可视化方案推荐嘛？

### DISCUSSION THREAD

**Discussion by RobertChaw** [NervJS/taro#10654] [Q & A]
Echarts 在 Taro3 里会有奇怪的错误不能正常使用，有其它的可视化方案推荐？

**Comment by aehyok** [comment]
目前可以运行的，[可下载代码预览](https://github.com/aehyok/taro-vue3-miniprogram)   
- https://github.com/aehyok/taro-vue3-miniprogram
